service/src/main/java/greencity/service/notification/NotificationServiceImpl.java

```diff
@@ -110,6 +137,23 @@ private UserNotification initialiseNotificationForUnpaidOrder(Order order, UserN
         return notification;
     }
 
+    private NotificationStep loadNotificationStep(NotificationStep step) {
+        if (step.equals(NotificationStep.ATTEMPT_1)) {
+            return NotificationStep.ATTEMPT_2;
+        }
+        if (step.equals(NotificationStep.ATTEMPT_2)) {
+            return NotificationStep.ATTEMPT_3;
+        }
+        return NotificationStep.ATTEMPT_1;
```

Comment:
I do not see any reason for `NotificationStep` class
If `step  ==  ATTEMPT_3` we will set it to `ATTEMPT_1` and do the same notification again and again
Should we block an order if it is not paid after 3 notifications?